src/math/noise.js

```diff
@@ -176,57 +252,54 @@ p5.prototype.noise = function(x, y = 0, z = 0) {
 };
 
 /**
+ * Adjusts the character of the noise produced by the
+ * <a href="#/p5/noise">noise()</a> function.
  *
- * Adjusts the character and level of detail produced by the Perlin noise
- * function. Similar to harmonics in physics, noise is computed over
- * several octaves. Lower octaves contribute more to the output signal and
- * as such define the overall intensity of the noise, whereas higher octaves
- * create finer-grained details in the noise sequence.
- *
- * By default, noise is computed over 4 octaves with each octave contributing
- * exactly half as much as its predecessor, starting at 50% strength for the 1st
- * octave. This falloff amount can be changed by adding an additional function
- * parameter. Eg. a falloff factor of 0.75 means each octave will now have
- * 75% impact (25% less) of the previous lower octave. Any value between
- * 0.0 and 1.0 is valid, however, note that values greater than 0.5 might
- * result in greater than 1.0 values returned by <a href="#/p5/noise">noise()</a>.
+ * Perlin noise values are computed over several octaves, similar to harmonics
```

Comment:
The phrase "computed over several octaves" might be unclear... Maybe explicitly stating that Perlin noise is created by adding together multiple layers of noise functions would be clearer?

I think the main idea I'm proposing is just to use the word "add" somewhere instead of "computed" which could be a little confusing.

Reply:
Good suggestion! Did my best to simplify–let me know how it's looking.